servicetalk-client-api/src/main/java/io/servicetalk/client/api/ServiceDiscovererEvent.java

```diff
@@ -19,15 +19,40 @@
 
 /**
  * Notification from the Service Discovery system that availability for an address has changed.
+ * <p>
+ * Interpreting Events
+ * <ul>
+ *     <li>When subscribing (or re-subscribing to recovery from faults) to an event stream the initial state is
```

Comment:
```
*     <li>When subscribing (or re-subscribing to recovery from faults) to an event stream the initial event is
 *         considered to be the current state of the world.
 * 
 * ```
 
 The "initial state is an empty set" is misleading, as if it meant the first event is that. I am not sure my suggestion is any better, just consider re-wording.